### PR TITLE
TestAcctlogRescUsedWithTwoMomHooks is not skipped appropriately

### PR DESCRIPTION
**Pull request by latha-subramanian**
#### Bug/feature Description
TestAcctlogRescUsedWithTwoMomHooks errors out when required number of moms are not provided via pbs_benchpress command 

#### Affected Platform(s)
All Linux

#### Cause / Analysis / Design
There is no check in the test to see if required number of moms are provided.

#### Solution Description
Test is skipped when required number of moms (2 moms) is not provided.

#### Testing logs/output
[beforefix.txt](https://github.com/PBSPro/pbspro/files/2432678/beforefix.txt)
[afterfix_nomoms.txt](https://github.com/PBSPro/pbspro/files/2432679/afterfix_nomoms.txt)
[afterfix_2moms.txt](https://github.com/PBSPro/pbspro/files/2432680/afterfix_2moms.txt)

#### Checklist:

- [x] I have joined the **[pbspro community forum](http://community.pbspro.org/)**.
- [x] My pull request contains a **single, signed** commit. See **[setting up gpg signature](https://pbspro.atlassian.net/wiki/display/DG/Signing+Your+Git+Commits)**.
- [x] My code follows the **[coding style](https://pbspro.atlassian.net/wiki/display/DG/Coding+Standards)** of this project.
- [ ] My change requires project documentation. See **[required documentation checklist](https://pbspro.atlassian.net/wiki/display/DG/Checklist+for+Developing+Features+and+Bug+Fixes)** for details.
   - [ ] I have added documentation in the **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)**.
- [ ] I have added new **PTL test(s) to my commit**. (See **[using PTL for testing](https://pbspro.atlassian.net/wiki/display/DG/Using+PTL+for+Testing)**) *(or)*
   - [ ] I have added  **manual test(s) to this pull request and explained why PTL is not appropriate** for this case.
- [ ] All new and existing automated tests have passed. (See **[running automated PTL tests](https://pbspro.atlassian.net/wiki/display/DG/PTL+Quick+Start+Guide)**).
- [x ] I have attached **test logs to this pull request** as evidence of testing/verification.

__***For further information please visit the [Developer Guide Home](https://pbspro.atlassian.net/wiki/display/DG/Developer+Guide+Home).***__

### CODE REVIEW

File: test/tests/functional/pbs_two_mom_hooks_resources_used.py [hunk 1]
```diff
@@ -47,6 +47,9 @@ class TestAcctlogRescUsedWithTwoMomHooks(TestFunctional):
 
     def setUp(self):
         TestFunctional.setUp(self)
+        if len(self.moms) != 2:
+            self.skipTest('Test requires two MoMs as input, ' +
+                          'use -p moms=<mom1>:<mom2>')
```
Comment (bhroam) [inline test/tests/functional/pbs_two_mom_hooks_resources_used.py]: Consider removing the plus.  It is slightly faster.  I'll sign off.

**Comment by vstumpf** [comment]
LGTM

**Comment by latha-subramanian** [comment]
Thanks @npadole20, @bhroam  and @vstumpf .
I have removed '+' from the skipTest message.
[afterfix.txt](https://github.com/PBSPro/pbspro/files/2435179/afterfix.txt)

**Comment by bhroam** [comment]
Looks good.  Please squash and rebase for the merge.

**Comment by latha-subramanian** [comment]
@bhroam , I have squashed and rebased. Ready for merge.

**Comment by bhroam** [comment]
I merged in the PR.

**Comment by latha-subramanian** [comment]
Thanks @bhroam !

**Pull request merged**